synapse/lib/view.py

```diff
@@ -459,14 +461,16 @@ async def runStorm():
                         [snap.on(n, chan.put) for n in show]
 
                     if shownode:
-                        async for pode in snap.iterStormPodes(text, opts=opts, user=user):
-                            await chan.put(('node', pode))
-                            count += 1
+                        with s_scope.enter(vals={'storm:query': s_common.guid()}):
+                            async for pode in snap.iterStormPodes(text, opts=opts, user=user):
+                                await chan.put(('node', pode))
+                                count += 1
 
                     else:
                         self.core._logStormQuery(text, user, opts.get('mode', 'storm'))
```

Comment:
Before we actually start indexing the query guids we could ad them into _logStormQuery in order to do some QA'ing